bin/test-sbt-1.0

```diff
@@ -2,7 +2,7 @@
 
 . "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/scriptLib"
 
-SCALA_VERSION="2.12.6"
+SCALA_VERSION="2.12.7"
```

Comment:
I'm a little on the fence about the update to 2.12.7.

We did it for Play 2.7.0-RC3, but we are on RC cycle and we hope to have 2.12.8 soonish.

Have you tried to build using 2.12.6 and akka 2.5.18?

Reply:
Hum, I would say that we can use 2.12.7 and if 2.12.8 happens before doing an RC for Lagom, then it should be an easy update.

Reply:
Now I see this was for 1.4.x. But I think this should be okay anyway.

Reply:
2.12.7 is better because it fixes the bad classifiles that 2.12.6 creates, which breaks Scala reflection.